### PR TITLE
Fixes #36602 - Mask hidden inputs in review step

### PR DESCRIPTION
**Pull request by adamruzicka**

### CODE REVIEW

File: webpack/JobWizard/steps/ReviewDetails/index.js [hunk 1]
```diff
@@ -122,11 +125,11 @@ const ReviewDetails = ({
       ),
       value: stringHosts(),
     },
-    ...templateInputs.map(({ name }) => ({
+    ...templateInputs.map(({ name, hidden_value: hidden }) => ({
```
Comment (ofedoren) [inline webpack/JobWizard/steps/ReviewDetails/index.js:128]:
```suggestion
    ...templateInputs.map(({ name, hidden_value: isHidden }) => ({
```

File: webpack/JobWizard/steps/ReviewDetails/index.js [hunk 2]
```diff
@@ -30,6 +30,9 @@ import { WizardTitle } from '../form/WizardTitle';
 import { parseEnd, parseRepeat } from './helpers';
 import { HostPreviewModal } from '../HostsAndInputs/HostPreviewModal';
 
+const maskValue = (hidden, value) =>
+  hidden ? '*'.repeat(value.length) : value;
```
Comment (ofedoren) [inline webpack/JobWizard/steps/ReviewDetails/index.js:34]:
```suggestion
const maskValue = (isHidden, value) =>
  isHidden ? '*'.repeat(value.length) : value;
```

File: webpack/JobWizard/steps/ReviewDetails/index.js [hunk 3]
```diff
@@ -122,11 +125,11 @@ const ReviewDetails = ({
       ),
       value: stringHosts(),
     },
-    ...templateInputs.map(({ name }) => ({
+    ...templateInputs.map(({ name, hidden_value: hidden }) => ({
       label: (
         <StepButton stepName={WIZARD_TITLES.hostsAndInputs}>{name}</StepButton>
       ),
-      value: templateValues[name],
+      value: maskValue(hidden, templateValues[name]),
```
Comment (ofedoren) [inline webpack/JobWizard/steps/ReviewDetails/index.js:132]:
```suggestion
      value: maskValue(isHidden, templateValues[name]),
```

**Review by ofedoren — COMMENTED** [review]
Just a suggestion:

File: webpack/JobWizard/steps/ReviewDetails/index.js [hunk 4]
```diff
@@ -122,11 +125,11 @@ const ReviewDetails = ({
       ),
       value: stringHosts(),
     },
-    ...templateInputs.map(({ name }) => ({
+    ...templateInputs.map(({ name, hidden_value: isHidden }) => ({
       label: (
         <StepButton stepName={WIZARD_TITLES.hostsAndInputs}>{name}</StepButton>
       ),
-      value: templateValues[name],
+      value: maskValue(isHidden, templateValues[name]),
```
Comment (MariaAga) [inline webpack/JobWizard/steps/ReviewDetails/index.js:132]: Should also be in `    ...advancedTemplateInputs.map(({ name }) => ({ value`

**Review by MariaAga — APPROVED** [review]

**Pull request merged**